package.json

```diff
@@ -52,7 +52,7 @@
   "devDependencies": {
     "coveralls-send": "0.0.2",
     "donejs": "*",
-    "donejs-cli": "^2.0.0-pre.3",
```

Comment:
we need to merge and release https://github.com/donejs/cli/pull/99 first

Reply:
I missed this change in my previous PR, maybe this was breaking the build, the generated app tests will fails without steal-conditional being installed.